backend/EasySpeak.RecommendationService/EasySpeak.RS.WebAPI/Controllers/RecommendationController.cs

```diff
@@ -0,0 +1,24 @@
+using EasySpeak.Core.Common.DTO.User;
+using EasySpeak.RS.WebAPI.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+
+namespace EasySpeak.RS.WebAPI.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class RecommendationController : ControllerBase
+{
+    private readonly IRecommendationService _recommendationService;
+
+    public RecommendationController(IRecommendationService recommendationService)
+    {
+        _recommendationService = recommendationService;
+    }
+
+    [HttpPost]
+    public async Task<List<long>> GetRecommendedUsers(NewRecommendationDto recommendationDto)
+    {
+        return await _recommendationService.GetRecommendedUsers(recommendationDto.ToDictionary());
```

Comment:
Again, can you return at least some recommendation score?
And when we don't have recommendation filter, it's also better to set some scores...